cmd/terramate/cli/project.go

```diff
@@ -225,25 +225,38 @@ func (p project) checkDefaultRemote() error {
 	)
 }
 
-func (p *project) checkLocalDefaultIsUpdated() error {
+func (p *project) checkRemoteDefaultBranchIsReachable() error {
 	gitcfg := p.gitcfg()
+
+	remoteDesc := fmt.Sprintf("remote(%s/%s)", gitcfg.DefaultRemote, gitcfg.DefaultBranch)
+
+	logger := log.With().
+		Str("HEAD", p.headCommit()).
+		Str("default_branch", remoteDesc).
+		Str("default_hash", p.remoteDefaultCommit()).
+		Logger()
+
+	outOfDateErr := errors.E(
+		ErrCurrentHeadIsOutOfDate,
```

Comment:
why do we have error messages somewhere else? what is the benefit?

Reply:
we use them to match errors in the e2e tests:
```
./cmd/terramate/e2etests/safeguard_test.go:81:			StderrRegex: string(cli.ErrCurrentHeadIsOutOfDate),
./cmd/terramate/e2etests/safeguard_test.go:148:				StderrRegex: string(cli.ErrCurrentHeadIsOutOfDate),
```